src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/CommunityAPIService.kt

```diff
@@ -29,8 +31,15 @@ interface CommunityAPIService {
     return "$description for $contractTypeName Referral ${referral.referenceNumber} with Prime Provider $primeProviderName\n$url"
   }
 
+  fun getNotes(referral: SentReferralDTO, url: String, description: String, contractTypeName: String, primeProviderName: String): String {
+    return "$description for $contractTypeName Referral ${referral.referenceNumber} with Prime Provider $primeProviderName\n$url"
+  }
+
   fun setNotifyPPIfRequired(appointment: Appointment) =
     NO == appointment.attended!! || appointment.notifyPPOfAttendanceBehaviour ?: false
+
+  fun setNotifyPPIfRequired(deliverySession: DeliverySessionDTO) =
```

Comment:
we can get rid of the null checking here and simplify to plain boolean expressions

```
  fun setNotifyPPIfRequired(appointment: Appointment): Boolean =
    appointment.attended == NO || appointment.notifyPPOfAttendanceBehaviour == true
 ```
 
 ```
  fun setNotifyPPIfRequired(deliverySession: DeliverySessionDTO): Boolean =
    deliverySession.sessionFeedback.attendance.attended == NO || deliverySession.sessionFeedback.behaviour.notifyProbationPractitioner == true
```